data/mods/Magiclysm/Spells/stormshaper.json

```
@@ -430,5 +431,461 @@
+    "//": "extra_effects for shocking_teleport, currently doesn't work, because of various issues in dash/teleport and extra_effects incompabilies.",
+    "valid_targets": [ "ally", "hostile", "ground" ],
+    "effect": "attack",
+    "effect_str": "zapped",
+    "shape": "blast",
+    "max_level": 25,
+    "min_damage": 10,
+    "max_damage": 85,
+    "damage_increment": 3,
+    "damage_type": "electric",
+    "flags": [ "LOUD" ],
+    "min_duration": 100,
+    "max_duration": 500,
+    "min_aoe": 1,
+    "max_aoe": 7,
+    "aoe_increment": 0.25,
+    "field_id": "fd_electricity",
+    "field_chance": 1,
+    "min_field_intensity": 1,
+    "max_field_intensity": 10,
+    "field_intensity_increment": 0.4,
+    "field_intensity_variance": 3
+  },
+  {
+    "id": "toxtricity",
+    "type": "SPELL",
+    "name": "Toxtricity",
+    "description": "Inject an electric energy into the creatures, which makes them weak, and deal huge damage over time.",
+    "effect": "attack",
+    "effect_str": "zapped",
+    "shape": "blast",
+    "valid_targets": [ "self", "ally", "hostile", "ground" ],
+    "flags": [ "SOMATIC", "VERBAL", "NO_LEGS" ],
+    "max_level": 25,
+    "min_damage": 0,
+    "max_damage": 13,
+    "damage_increment": 0.5,
+    "min_aoe": 1,
+    "max_aoe": 11,
+    "aoe_increment": 0.4,
+    "min_range": 6,
+    "max_range": 21,
+    "range_increment": 0.6,
+    "min_duration": 0,
+    "max_duration": 1500,
+    "duration_increment": 60,
+    "base_casting_time": 350,
+    "final_casting_time": 150,
+    "casting_time_increment": -8,
+    "base_energy_cost": 400,
+    "final_energy_cost": 125,
+    "energy_increment": -11,
+    "energy_source": "MANA",
+    "difficulty": 5,
+    "spell_class": "STORMSHAPER",
+    "min_dot": 0,
+    "max_dot": 15,
+    "dot_increment": 0.6,
+    "damage_type": "electric"
+  },
+  {
+    "id": "electric_eye",
+    "type": "SPELL",
+    "name": "Electric Eye",
+    "description": "Enhance your sight with your powers.  It allows you to see creatures of the same energy around you, and some minor bonuses.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "effect_str": "electric_eye",
+    "shape": "blast",
+    "flags": [ "NO_LEGS", "SILENT", "NO_EXPLOSION_SFX" ],
+    "skill": "deduction",
+    "spell_class": "STORMSHAPER",
+    "energy_source": "MANA",
+    "difficulty": 4,
+    "base_casting_time": 3000,
+    "final_casting_time": 1000,
+    "casting_time_increment": -80,
+    "base_energy_cost": 400,
+    "final_energy_cost": 200,
+    "energy_increment": -8,
+    "max_level": 25,
+    "min_duration": 57600,
+    "max_duration": 1440000,
+    "duration_increment": 57600
+  },
+  {
+    "id": "electric_waves",
+    "type": "SPELL",
+    "name": "Electric Waves",
+    "description": "Your hands start to emit a huge amount of energy, that you release with each your attack.  The damage is increases the more you master the spell.",
```

```suggestion
    "description": "Your hands start to emit a huge amount of energy, that you release with each attack.",
```
Players can see damage scaling when they study the spell, removed redundancy.

---

Thats the problem - player cant see. The effect works using enchantment system, and now we have no way to show information about what effect cause `effect` and how big the effect is